vhdbuilder/packer/vhd-scanning.sh

```diff
@@ -3,6 +3,29 @@ set -eux
 
 source ./parts/linux/cloud-init/artifacts/cse_benchmark_functions.sh
 
+if [ -z "$1" ]; then
```

Comment:
nit: can we rename positional args (other than $0) before usage for better readability?

Reply:
This is actually from another test run that I did. I will remove them.